srml/im-online/src/lib.rs

```diff
@@ -532,7 +533,7 @@ impl<T: Trait> support::unsigned::ValidateUnsigned for Module<T> {
 			}
 
 			Ok(ValidTransaction {
-				priority: 0,
+				priority: TransactionPriority::max_value(),
```

Comment:
is there anything to stop validators from issuing _too many_ heartbeats?

Reply:
Had the same question. My second thought was that this might be a common problem among operational transactions.

Reply:
Since `provides` tag is `(current_session, authority_id)` there can exist only one such transaction in the pool. The authority could spam transactions from different sessions (or from different `authority_id`) but they should be rejected through validation. So there can only be as much heartbeats in the pool as there is authorities. If the heartbeat is included in block it should invalidate duplicates from being included.

Reply:
OK, thanks for the clarification.